src/ccache.cpp

```diff
@@ -17,7 +17,9 @@
 // this program; if not, write to the Free Software Foundation, Inc., 51
 // Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
 
+// clang-format off
```

Comment:
how can a simple include be formatted incorrectly?

Reply:
It gets reordered into the block below, another option is to put a single comment in-between.

Reply:
That issue only seems to affect some versions of `clang-format`, but I changed it to a less ugly fix in any case.